schema-admin.graphql

```diff
@@ -205,6 +198,10 @@ type ApprovedCorpusItem @key(fields: "url") {
     A single sign-on user identifier of the user who last updated this entity. Null on creation.
     """
     updatedBy: String
+    """
+    Subquery to get the ScheduledCorpusItems to display to a user for a given date
+    """
+    scheduledSurfaceHistory(scheduledSurfaceGuid: ID, limit: Int): [ScheduledCorpusItem!]!
```

Comment:
Note that both filters are optional to future-proof this query. Right now the business case is to fetch this data for a single scheduled surface id + just the one most recent result, but eventually we will need to show on the frontend the entire history for a curated corpus item on demand, and that should show scheduled entries for all surfaces - either all or 6? 12? most recent ones.

Reply:
Wondering if the result returned from this should be a `nullable` value? i.e `[ScheduledCorpusItem!]`, there could be a scenario where it hasn't been scheduled yet for the first time so this shouldn't return any history

Reply:
Agree these would be needed if we left the original `ScheduledCorpusItem` as a return type in place, but after a discussion with @Herraj and also feedback from @bassrock further down the review comments we decided a custom type that doesn't contain approved items would serve us better here.

Reply:
@Herraj - adding here for posterity as we discussed this in a pairing earlier today - the non-nullable return is for the frontend to have an easier time with return types. Something is always guaranteed to be returned, even if it's just an empty array.

Reply:
@nina-py I think I was unclear. I meant to imply return ScheduledCorpusItem but we can limit the query depth so a user can't infinitely recurse.

Reply:
also just seeing the return type comment for frontend. makes sense